presto-manager-common/src/main/java/com/teradata/prestomanager/common/ApiRequester.java

```diff
@@ -48,7 +48,7 @@ private ApiRequester(Client client,
             String mediaType)
     {
         this.client = requireNonNull(client);
-        this.uriTemplate = requireNonNull(uriBuilder).toTemplate();
+        this.uriTemplate = new UriCopyBuilder(requireNonNull(uriBuilder));
```

Comment:
It's not needed because the c-tor of `UriCopyBuilder` checks for nullness already.